pkg/services/pluginsintegration/clientmiddleware/logger_middleware.go

```diff
@@ -37,43 +38,50 @@ func (m *LoggerMiddleware) logRequest(ctx context.Context, pluginCtx backend.Plu
 	status := statusOK
 	start := time.Now()
 	timeBeforePluginRequest := log.TimeSinceStart(ctx, start)
+
+	ctx = instrumentContext(ctx, endpoint, pluginCtx)
 	err := fn(ctx)
 	if err != nil {
 		status = statusError
 		if errors.Is(err, context.Canceled) {
 			status = statusCancelled
 		}
 	}
-
-	logParams := []any{
+	params := logParams(pluginCtx)
+	params = append(params,
+		"endpoint", endpoint,
 		"status", status,
 		"duration", time.Since(start),
-		"pluginId", pluginCtx.PluginID,
-		"endpoint", endpoint,
 		"eventName", "grafana-data-egress",
 		"time_before_plugin_request", timeBeforePluginRequest,
+	)
+	if traceID := tracing.TraceIDFromContext(ctx, false); traceID != "" {
+		params = append(params, "traceID", traceID)
 	}
-
-	if pluginCtx.User != nil {
-		logParams = append(logParams, "uname", pluginCtx.User.Login)
-	}
-
-	traceID := tracing.TraceIDFromContext(ctx, false)
-	if traceID != "" {
-		logParams = append(logParams, "traceID", traceID)
+	if status == statusError {
+		params = append(params, "error", err)
 	}
+	m.logger.Info("Plugin Request Completed", params...)
```

Comment:
Nit. Using `m.Logger.FromContext(ctx)` would automatically give you traceId. However it would also include all params from `instrumentContext` so maybe not optimal. Maybe something to revisit later.